pkg/catalog/debugger.go

```diff
@@ -10,10 +10,10 @@ import (
 
 // ListSMIPolicies returns all policies OSM is aware of.
 func (mc *MeshCatalog) ListSMIPolicies() ([]*split.TrafficSplit, []identity.K8sServiceAccount, []*spec.HTTPRouteGroup, []*access.TrafficTarget) {
```

Comment:
Optional: we could remove the ListSMIPolicies method, and have the single calling site, call each of the below methods individually. Up to you